sdk/storage/storage-blob/review/storage-blob.api.md

```diff
@@ -1013,6 +1063,21 @@ export interface BlobSyncCopyFromURLOptions extends CommonOptions {
     metadata?: Metadata;
     sourceConditions?: ModifiedAccessConditions;
     sourceContentMD5?: Uint8Array;
+    tags?: BlobTags;
+}
+
+// @public
+export interface BlobTag {
+    // (undocumented)
+    key: string;
+    // (undocumented)
+    value: string;
+}
+
+// @public
+export interface BlobTags {
+    // (undocumented)
+    blobTagSet: BlobTag[];
```

Comment:
Can this be `tags`? `blobTagSet` is pretty verbose.

Reply:
In the resulting API there's a lot of `tags.blobTagSet`. Even if it's `tags.tags`, the duplication is still annoying. It would be nice if BlobTags was simply `BlobTag[]` but the issue with this approach is you end up having data properties hanging off of an array for e.g. the members of `GetBlobTagHeaders`. If these are very uncommonly used, this might be ok. Was this approach considered, and do you have any thoughts here?

Reply:
Is it allowed to have two entries with the same key in the array?

Reply:
Interface `BlogTags` and `blobTagSet: BlobTag[]` are generated types.

@jeremymeng Service should not allow duplicate keys. 

I'd like to define `Tags` type like following. This is more align with other lanauges which use Dictionary for tags type.
```javascript
export interface Tags {
  [propertyName: string]: string;
}
```

Reply:
Updated to `Tags` type everywhere in options and responses. For example,
```javascript
    const tags = {
      tag1: "val1",
      tag2: "val2"
    };
    await blockBlobClient.setTags(tags);
    const response = await blockBlobClientSnapshot.getTags();
    assert.deepStrictEqual(response.tags, tags);
```